p2p/src/net/default_backend/backend.rs

```diff
@@ -58,6 +58,9 @@ struct PeerContext {
     /// Note that sending may fail unexpectedly if the connection is closed!
     /// Do not propagate ChannelClosed error to the higher level, handle it locally!
     tx: mpsc::UnboundedSender<Event>,
+
+    /// True if the peer was accepted by PeerManager and SyncManager was notified
+    was_accepted: bool,
```

Comment:
While boolean looks to be valid, but it's the wrong representation of this value. We should use a flag, where once it's set, it cannot be unset. Isn't this what we're trying to represent, here?

Reply:
Yes, that's right. But do we have such type? I can't find anything.

Reply:
Either create it, or throw in a TODO, explaining this problem.

If you're going with a TODO, please explain the problem fully. We have a problem of very short and not-understandable TODO some times.

Reply:
That's understandable. It's a best-effort technique.